src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java

```diff
@@ -39,7 +39,7 @@
 import appeng.core.AppEng;
 
 
-public class ModelOverrideComponent implements PreInitComponent
+public class ModelOverrideComponent implements IPreInitComponent
 {
 
 	private static final ModelResourceLocation MODEL_MISSING = new ModelResourceLocation( "builtin/missing", "missing" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "MODEL_MISSING" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)